examples/nvidia/specify_card_type_not_use.yaml

```diff
@@ -3,8 +3,10 @@ kind: Pod
 metadata:
   name: gpu-pod
   annotations:
+    # You can run command: kubectl get node $node -o jsonpath='{.metadata.annotations.hami\.io/node-nvidia-register}' to get registered gpu info
+    # GPU type full name is like NVIDIA-NVIDIA A100,short name is like A100
```

Comment:
Missing a white space after comma.
I think it would be better to be "The full GPU type name is like NVIDIA-NVIDIA A100, while the short name is like A100"